src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+        }
+    }
+
+    /**
+     * Creates a new Collection that has each entry numbered in ascending order
+     */
+    private static void printAll() {
+        List<String> printables = new ArrayList<>();
+        printables.add(LISTMESSAGE);
+        for (int i = 0; i < list.size(); i++) {
+            Task task = list.get(i);
+            int index = i + 1;
+            printables.add(String.format("%d.%s", index, task.toString()));
+        }
+        prettyPrint(printables);
+    }
+
+    /**
+     * Printing empty list
+     */
+    private static void prettyPrint() {
+        prettyPrint("");
+    }
+
+    /**
+     * Printing when everything has already been formatted into a single entry
+     * @param printable
+     */
+    private static void prettyPrint(String printable) {
+        String out = String.format("%s\n%s %s\n%s", LINE, TAB, printable, LINE);
+        System.out.println(out);
+    }
+
+    /**
+     * Printing when items are still in a List.
+     * @param printables
+     */
+    private static void prettyPrint(List<String> printables) {
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < printables.size(); i++) {
+            String s = printables.get(i);
+            if (i == 0) {
+                sb.append(s);
+            }
+            else {
+                sb.append(String.format("\n%s %s", TAB, s));
+            }
+        }
+        String printable = sb.toString();
+        prettyPrint(printable);
+    }
+
+    public static void main(String[] args) throws DukeException {
+        Scanner sc = new Scanner(System.in);
+        prettyPrint(WELCOMEMESSAGE);
+        boolean canNext = true;
+        while (canNext) {
+            try {
+                String inputCmd = sc.next();
+                String inputRem = sc.nextLine();
+                switch (inputCmd) {
+                    case (ENDCOMMAND):
+                        canNext = false;
+                        break;
+                    case (PRINTCOMMAND):
+                        printAll();
+                        break;
+                    case (MARKCOMMAND):
+                        markTask(inputRem);
+                        break;
+                    case (UNMARKCOMMAND):
+                        unmarkTask(inputRem);
+                        break;
+                    case (TODOCOMMAND):
+                        createTask(inputRem, TaskType.TODO);
+                        break;
+                    case (DEADLINECOMMAND):
+                        createTask(inputRem, TaskType.DEADLINE);
+                        break;
+                    case (EVENTCOMMAND):
+                        createTask(inputRem, TaskType.EVENT);
+                        break;
+                    case (DELETECOMMAND):
+                        deleteTask(inputRem);
+                        break;
+                    default: throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
```

Comment:
I see that you're making good use of switch statements! Consider the following requirements:
`The switch statement should have the following form: Note there is no indentation for case clauses.`